discovery-provider/docker-compose.yml

```diff
@@ -109,6 +109,21 @@ services:
       - AUTOHEAL_INTERVAL=10
       - CURL_TIMEOUT=30
 
+  network-monitoring-push-gateway:
+    image: prom/pushgateway
+    container_name: pushgateway
+    ports:
+      - 9091:9091
+    networks:
+      - discovery-provider-network
+  
+  network-monitoring:
+    image: audius/network_monitoring
+    container_name: network_monitoring
+    depends_on:
+      - network-monitoring-db
```

Comment:
This does not exist 👀

Reply:
I know, that's why I wanna hop on a call - this is still a draft PR